### PR TITLE
Issue108 support python versions

### LINKED ISSUE #108: Update tests to support Python 3.10
- [ ] Python 3.10

(see also #114 for supporting 3.11)

### PR DESCRIPTION
**Pull request by jo-asplin-met-no**
**Summary**: Support Python versions 3.6 ... 3.10 in the github pipeline

**Related issue**: closes #108 

**Suggested reviewer(s)**: mortenwh

**Reviewer checklist**:

* [ ] The headers of all files contain a reference to the repository license
* [ ] 100% test coverage of new code - meaning:
  * [ ] The overall test coverage increased or remained the same as before
  * [ ] Every function is accompanied with a test suite
  * [ ] Tests are both positive (testing that the function work as intended with valid data) and negative (testing that the function behaves as expected with invalid data, e.g., that correct exceptions are thrown)
  * [ ] Functions with optional arguments have separate tests for all options
* [ ] Examples are supported by doctests
* [ ] All tests are passing
* [ ] All names (e.g., files, classes, functions, variables) are explicit
* [ ] Documentation (as docstrings) is complete and understandable

The checklist is based on the S-ENDA conventions and definition of done (see [General Conventions](https://s-enda-documentation.readthedocs.io/en/latest/general_conventions.html)). The above points are not necessarily relevant to all contributions. In that case, please add a short explanation to help the reviewer.

### CODE REVIEW

File: .github/workflows/pytest.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ jobs:
   pyTestCov:
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.6, 3.7, 3.8, 3.9,'3.10']
```
Comment (mortenwh) [inline .github/workflows/pytest.yml:13]: hvorfor `'3.10'` og ikke bare `3.10`?

**Review by mortenwh — APPROVED** [review]
Ser bra ut - lurer bare på en liten detalj :)

File: .github/workflows/pytest.yml (hunk 1 above)
Comment (johtoblan) [reply .github/workflows/pytest.yml:13]: 3.10 parses som 3.1 i yaml. Python 3.1 skal vi ikke støtte

Comment (johtoblan) [reply .github/workflows/pytest.yml:13]: Så '3.10' er eneste måte å få yaml-filen til å representere 3.10

**Pull request merged**